docs/topics/api/accounts.rst

```diff
@@ -72,7 +72,65 @@ This endpoint is a shortcut to your own account. It returns an :ref:`account obj
 
 .. http:get:: /api/v3/accounts/profile/
 
-    .. _self-account-object:
+
+----------------
+Collections List
```

Comment:
I did originally have collections as a separate endpoint (see commit history) but with user always being required it seemed redundant so I moved it under accounts.